extension-style-app/src/androidTest/java/com/mapbox/maps/testapp/style/sources/generated/GeoJsonSourceTest.kt

```diff
@@ -43,6 +43,14 @@ class GeoJsonSourceTest : BaseStyleTest() {
     assertEquals(null, testSource.data)
   }
 
+  @Test
+  @UiThreadTest
+  fun emptyDataTest() {
+    val testSource = geoJsonSource("testId")
+    setupSource(testSource)
+    assertEquals(null, testSource.data)
```

Comment:
nit: testSource.data is should always return null

Reply:
What we can verify is that there's exceptions while setup the GeoJsonSource without specifying the data.

Reply:
@ank27 Please also trigger the `ci/circleci: build/start-run-style-test` manually to verify the instrumentation tests, since the change is related to the style extension.

Reply:
done and verified all test cases. :)